src/model/model_json.nit

```diff
@@ -166,6 +166,7 @@ end
 redef class MClassDef
 	redef fun json do
 		var obj = super
+		obj["mdoc"] = mdoc
```

Comment:
I agree with @privat 

But this will negatively affect nitweb where a lot of field will be blank without the fallback.

I suggest to perform the change as proposed by @privat and to monkey patch nitweb. I can do the refactoring later.